.azure-pipelines/gpg.yml

```diff
@@ -1,5 +1,12 @@
 
 parameters:
+- name: name
```

Comment:
nit: rather than name, i think downloadTaskName or something would be better.

Reply:
i made it `nameDownloadTask` for consistency and becasue `downloadKey` is saying download the key whereas `downloadTaskName` is not downloading anything